src/components/StaticImgCloseup.tsx

```diff
@@ -1,24 +1,29 @@
-import useImage from '../hooks/useImage';
-import { useSelector } from 'react-redux';
+import useImage from "../hooks/useImage";
+import { useSelector } from "react-redux";
 import { RootState } from "../app/store";
-
+import { ShowIf } from "./ShowIf";
 
 const StaticImgCloseup: React.FC = () => {
-
-    const flyToId = useSelector((state: RootState) => state.app.flyToId);
-    const currentStaticImg = useSelector((state: RootState) => state.app.currentStaticImg);
-
-    const imageName = flyToId + '_sat_' + currentStaticImg;
-    const {image} = useImage(imageName);
-
-    
-    return (
-
-            <div >
-            <img src={image} id={imageName} alt={imageName} />
-            </div>
-
-    )
-}
-
-export default StaticImgCloseup
\ No newline at end of file
+  const flyToId = useSelector((state: RootState) => state.app.flyToId);
+  const currentStaticImg = useSelector(
+    (state: RootState) => state.app.currentStaticImg
+  );
+
+  const imageName = flyToId + "_sat_" + currentStaticImg;
+  const { image, loading } = useImage(imageName);
+
+  return (
+    <ShowIf value={!loading}>
+      <div className="static_satellite_image_container">
+        <img
+          src={image}
+          id={imageName}
+          alt={imageName}
+          className="static_satellite_closeup"
+        />
+      </div>
+    </ShowIf>
```

Comment:
**Not Loading Images: BugFix AKDS-100**
- I am not certain if this is the problem, but in testing on staging in the inspect tool I did notice those flashes of broken images before they load.
- Perhaps the browser has a better recovery than the phone.
- So I added the `ShowIf` to see if that helps.

**Uncentered Static Images: AKDS-99**
- For centering the static-images on multi width devices, with the `static-satellite-closeup` images being position absolute and manually adjusting the images left and top was only making it centered on some device widths and not others.
- To fix this I targeted the div, made it a flex container of `position-absolute`, `width: 100vw` and flex centered the `img` inside of it. 
- This way it is always centered at every width.